### PR TITLE
Add blog post about fastText models

### PR DESCRIPTION
**Pull request by sheonhan**
Add blog post about fastText word vectors and language identification model integration
https://sheonhan-obscure-chainsaw-r64pvgqgw7fxx5q-5564.preview.app.github.dev/blog/fasttext

### CODE REVIEW

File: _blog.yml [hunk 1]
```diff
@@ -2258,3 +2258,13 @@
     - community
     - guide
     - game-dev
+
+- local: fasttext
+  title: "Welcome fastText to the 🤗 Hub"
+  author: sheonhan
+  thumbnail: /blog/assets/147_fasttext/thumbnail.png
```
Comment (julien-c) [inline _blog.yml:2265]: the thumbnail you can make it lighter-weight (file size) with https://tinypng.com/

**Review by julien-c — COMMENTED** [review]
looks good so far!

**Review by julien-c — APPROVED** [review]
lgtm

**Comment by julien-c** [comment]
@sheonhan i can merge, do you want me to merge now?

**Comment by sheonhan** [comment]
@julien-c Yes please! I'm still waiting for my PR for updating the official documentation to be approved, so hopefully I get an approval there soon https://github.com/facebookresearch/fastText/pull/1335

**Pull request merged**